graphics/tinygl/init.cpp

```diff
@@ -44,7 +44,15 @@ void initSharedState(GLContext *c) {
 void endSharedState(GLContext *c) {
 	GLSharedState *s = &c->shared_state;
 
-	free_texture(c, 0);
```

Comment:
I removed that commit, I'll see if I can get somewhere changing import dependencies...